src/router/index.ts

```diff
@@ -11,6 +12,11 @@ const routes: RouteRecordRaw[] = [
     name: 'Main',
     component: Main
   },
+  {
+    path: '/shiden',
```

Comment:
I recommend that we auto direct the active crowdloan page when the user enters from the root dir.
So the Polkadot crowdloan should be `/astar` and Kusama be `/shiden` and we switch the current active crowdloan to be redirected from `/`

Reply:
Fixed